4-AdvancedGrants/2-call-api-api-ca/DownstreamAPI/config.json

```diff
@@ -0,0 +1,16 @@
+{
+    "credentials": {
+        "tenantID": "Enter_the_Tenant_Id_Here",
+        "clientID": "Enter_the_Application_Id_Here"
```

Comment:
>clientID": "Enter_the_Application_Id_Here" [](start = 9, length = 42)

comma added after client id breaks the code

Reply:
This is something I need to fix in the Codegenerator :/